### DISCUSSION TITLE
「リスト表示」でタイトル横に表示されるアイコンの挙動

### DISCUSSION THREAD

**Discussion by hide-o-mvd** [growilabs/growi#7744] [Ideas / Requests] [answered]
Environment
------------

### Host

| item     | version |
| ---      | --- |
|OS        |Debian 4.9.110|
|GROWI     |3.5.23|
|node.js   |10.17.0|
|npm       |6.11.3|
|Using Docker|yes|
|Using [growi-docker-compose][growi-docker-compose]|yes|

[growi-docker-compose]: https://github.com/weseek/growi-docker-compose

### Client

| item     | version |
| ---      | --- |
|OS        | Windows10|
|browser   |Google Chrome 79.0.3945.88|

How to reproduce? (再現手順)
---------------------------

1. 任意の記事にアクセスする
1. 記事の投稿者と別のユーザがコメントする
1. トップに戻り、記事リストを確認する

What happens? (症状)
---------------------
記事タイトルの左にあるアイコンに、コメント投稿者のアイコンが表示されている。

What is the expected result? (期待される動作)
-------------------------------------------
記事の最終更新者のアイコンを表示したままにしてほしい。

Note
----
コメント投稿しただけでも記事内容を更新したと勘違いされてしまうため、コメント投稿ではアイコンが変わらないほうが良いなと感じております。
現状のほうが望ましいという考えもあると思いますので、一要望として捉えていただければ幸いです。

**Comment by yuki-takei** [comment]
開発チーム内で先週検討を行いました。

[/GROWI村議議事録/20200227](https://dev.growi.org/5e5751ad1337da0048e5fc57)

アイコンにどのユーザーが出てきて欲しいのかは、運用現場の好みに応じて3種類ありそうです。

* ページ作成者固定
* ページコンテンツの最終更新者
* ページコンテンツの最終更新者 または コメントの最終投稿者

ゴールとしては、これらを管理画面で選択可能にする予定です。
着手はまだ先になりそうですが、今しばらくお待ちいただければと思います。

**Comment by yuki-takei** [comment, marked as answer]
こちら現時点の仕様とし、クローズします。
改修を希望する方がいらっしゃった場合はコメントと reopen をお願いします。
